sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_mcp.py

```diff
@@ -0,0 +1,169 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the
+    Model Context Protocol (MCP) tool from the Azure Agents service using a synchronous client.
+    To learn more about Model Context Protocol, visit https://modelcontextprotocol.io/
+
+USAGE:
+    python sample_agents_mcp.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-ai-agents azure-identity
```

Comment:
Let's add `--pre` because this requires beta version of azure-ai-agents and without this option stable version gets installed.